numba/core/analysis.py

```diff
@@ -402,6 +402,15 @@ def resolve_input_arg_const(input_arg_idx):
         print("before".center(80, '-'))
         print(func_ir.dump())
 
+    phi2lbl = dict()
+    phi2asgn = dict()
+    for lbl, blk in func_ir.blocks.items():
+        for stmt in blk.body:
+            if isinstance(stmt, ir.Assign):
+                if isinstance(stmt.value, ir.Expr) and stmt.value.op == 'phi':
+                    phi2lbl[stmt.value] = lbl
+                    phi2asgn[stmt.value] = stmt
+
```

Comment:
nitpicking: any reason for this block of code to be so far away from the use of `phiasgn`?